packages/web-react/src/components/Stack/README.md

```diff
@@ -1,6 +1,45 @@
 # Stack
 
-Use Stack to vertically space content. It works with lists or any other HTML elements.
-It can be used to separate form fields, for example.
+[Stack] is a component that allows you to compose elements vertically.
 
-For detailed information see [Stack](https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web/src/scss/components/Stack/README.md) component
+Basic example usage:
+
+```jsx
+<Stack>
+  <div>Block 1</div>
+  <div>Block 2</div>
+  <div>Block 3</div>
+</Stack>
+```
+
+Advanced example usage:
+
+```jsx
+<Stack elementType="ul" hasMiddleDividers hasTopDivider hasBottomDivider>
+  <li>
+    <div>List item 1</div>
+  </li>
+  <li>
+    <div>List item 1</div>
+  </li>
+  <li>
+    <div>List item 1</div>
+  </li>
+</Stack>
+```
+
+## API
+
+| Prop name           | Type     | Default | Required | Description                          |
+| ------------------- | -------- | ------- | -------- | ------------------------------------ |
+| `class`             | `string` | `null`  | no       | Custom CSS class                     |
+| `elementType`       | `string` | `div`   | no       | Element type of the wrapper element  |
+| `hasBottomDivider`  | `bool`   | `false` | no       | Render a divider after a last item   |
```

Comment:
What about having also a `hasDividers` prop that accepts an object, e.g. it could be easier to set up dividers once in one prop using an object than setting up 3 props. This is an extension so I do not suggest removing of existing divider props.

Reply:
I'm not sure about that. Seems a little bit like overengineering. It would use another way to pass these props than Twig implementation.